amenity-points.mss

```diff
@@ -580,56 +585,91 @@
     marker-fill: @shop-icon;
   }
 
-  [feature = 'shop_art'][zoom >= 17] {
-    marker-file: url('symbols/shop/art.svg');
+  [feature = 'shop_art'][zoom >= 18] {
+    [zoom < 18] {
+      marker-width: 4;
+    }
+    [zoom >= 18] {
+      marker-file: url('symbols/shop/art.svg');
+    }
     marker-fill: @shop-icon;
     marker-placement: interior;
     marker-clip: false;
   }
 
-  [feature = 'shop_bag'][zoom >= 17] {
+  [feature = 'shop_bag'][zoom >= 18] {
     marker-file: url('symbols/shop/bag.svg');
     marker-fill: @shop-icon;
     marker-placement: interior;
     marker-clip: false;
   }
 
-  [feature = 'shop_bakery'][zoom >= 17] {
-    marker-file: url('symbols/shop/bakery.svg');
+  [feature = 'shop_bakery'][zoom >= 18] {
+    [zoom < 18] {
+      marker-width: 4;
```

Comment:
I think this misses `marker-line-width: 0` to avoid the darker outlines that can be seen in the screenshot.